systest/tests/timeskew_test.go

```diff
@@ -0,0 +1,60 @@
+package tests
+
+import (
+	"context"
+	"testing"
+
+	spacemeshv1 "github.com/spacemeshos/api/release/go/spacemesh/v1"
+	"github.com/spacemeshos/go-spacemesh/systest/chaos"
+	"github.com/spacemeshos/go-spacemesh/systest/cluster"
+	"github.com/spacemeshos/go-spacemesh/systest/testcontext"
+	"golang.org/x/sync/errgroup"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestShortTimeskew(t *testing.T) {
+	const (
+		enableSkew = 10
+		stopSkew   = enableSkew + 3
+		stopTest   = stopSkew + 3
+		skewOffset = "-3s" // hare round is 2s
+	)
+	tctx := testcontext.New(t, testcontext.Labels("sanity"))
+	cl, err := cluster.Default(tctx, cluster.WithKeys(10))
+	require.NoError(t, err)
+
+	failed := int(0.2 * float64(tctx.ClusterSize))
+	eg, ctx := errgroup.WithContext(tctx)
+	client := cl.Client(0)
+	scheduleChaos(ctx, eg, client, enableSkew, stopSkew, func(ctx context.Context) (chaos.Teardown, error) {
```

Comment:
can you add comments roughly about what this function `scheduleChaos` do?

Reply:
i mean how does this function cause hare to have the two failure scenarios below?

Reply:
there was a doc about chaos.Timeskew, i updated a comment in the test as well

Reply:
is it possible that 2 hare committees are formed, one from the 80% and one from the 20% skewed? don't know hare that well...

Reply:
i think this is possible if they received different sets of activations, e.g. there was a long network partition